Editor/MeshChainShaderGUI.cs

```diff
@@ -1,12 +1,12 @@
-﻿using UnityEngine;
+﻿using UnityEditor;
+using UnityEngine;
 
-namespace UnityEditor
+namespace Unity.Labs.XR
```

Comment:
@AndrewTHEManeri does this make sense for a namespace? I had originally used `Unity.Labs.XRLineRenderer`, but that creates conflicts in classes that use the `XRLineRenderer` class itself. It's the same reason why we can't use the namespace `Unity.Labs.MARS.Editor` since it conflicts with `UnityEngine.Editor` and why `ModuleLoaderCore` has the `Core` suffix, so that it can live in a namespace called `Unity.Labs.ModuleLoader`.

Reply:
Short answer - Yah
Longer answer - I dislike when the namespace has the name of the class, it can be very redundant.  In terms of a Unity.Labs namespace, my big concern there is people are going to see the word 'Labs' and associate that with 'Experimental'